CRM/Financial/BAO/FinancialAccount.php

```diff
@@ -49,7 +49,7 @@ public function __construct() {
    *
    * @return CRM_Financial_BAO_FinancialAccount
    */
-  public static function retrieve(&$params, $defaults = []) {
+  public static function retrieve(&$params, &$defaults) {
```

Comment:
you can still  have  = [] even  when  passed by reference  - that might cover both issues?